packages/angular_devkit/core/package.json

```diff
@@ -7,6 +7,23 @@
   "keywords": [
     "core"
   ],
+  "exports": {
+    ".": {
+      "default": "./src/index.js",
+      "types": "./src/index.d.ts"
```

Comment:
The [docs for conditional exports](https://nodejs.org/api/packages.html#conditional-exports) specifies that `default` should always come last.